web-console/knikubevirt/list-view-pf4/list-view-pf4.md

```diff
@@ -0,0 +1,92 @@
+# List View (updated)
+
+## VM List
+
+The VM list contains columns for the following:
+
+* Name
+* Namespace
+* Status
+  * Running, Off, Error (with custom messages), Cloning, Migrating, Importing, Warning, Pending (with changes that will apply after restart), Unknown
+* Created
+  * Date formatting should follow OpenShift’s console
+  * If possible, a tooltip should appear on hover with the exact creation date, including year.
+* Node
+* IP Address
+
+### Max number of columns and prioritization
+
+We set 6 columns to be the max number we will display in list views. We also determined priorities/usefulness of each column and set the priority order from left to right.
+
+![VM list six columns](img/vmlist.png)
+
+### Status Popovers and Icons
+
+[The Resource status](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/status/status)
+
+Statuses should be colored blue to indicate that they can be clicked/tapped.
+Hovering should only underline the blue status text.
+Hovering over a blue status should display a status popovers with a brief description of the issue or underlying process.
+Clicking on the "Learn more" with the external link icon (seen below in the Guest Agent example) should bring the user to external documentation for further details.
+“View details” without the external link icon would be more appropriate if the status is an error, for example, and clicking would bring the user to the Events page.
+The status popovers are replacing the status tooltips  because they're significantly more accessible on mobile and easier to copy/paste from and they also provide actions that the user can take upon.
```

Comment:
```suggestion
The status popovers are replacing the status tooltips because they're significantly more accessible on mobile and easier to copy/paste from and they also provide actions that the user can take upon.
```